src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -116,6 +116,17 @@
                                         </StackPanel>
                                     </labs:SettingsExpander.Header>
                                     <labs:SettingsExpander.Content>
+                                    <StackPanel Orientation="Horizontal">
+                                        <FontIcon
+                                            Margin="0,0,8,0"
+                                            Glyph="&#xE7BA;"
+                                            Foreground="Yellow"
```

Comment:
Is this visible in light mode? If not we should bind against a theme resource for the `InfoBarIconBackgrond`. You can look at the PT Run plugin list code for an example on how to do it.

Reply:
Good catch!

Reply:
```
Foreground="{ThemeResource InfoBarWarningSeverityIconBackground}"
```

Reply:
Thank you! :) Very helpful!